azdev/operations/coverage/__init__.py

```diff
@@ -0,0 +1,199 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+import os
+import sys
+
+from knack.util import CLIError
+from knack.log import get_logger
+
+from azdev.utilities import (display, get_path_table, require_azure_cli, filter_by_git_diff)
+from azdev.utilities.path import get_cli_repo_path
+
+from azdev.operations.linter.util import filter_modules
+
+logger = get_logger(__name__)
+
+
+def test_coverage_cli(modules=None, git_source=None, git_target=None, git_repo=None,
+                      include_whl_extensions=False, save_global_exclusion=False,
+                      cli_ci=False):
+    import mock
+    exclusion_path = os.path.join(*[get_cli_repo_path(), 'test_exclusions.json'])
+    exit_code = 0
+
+    with mock.patch('azure.cli.core.commands.validators.validate_file_or_dict', return_value={}),\
+            mock.patch('azure.cli.core.util.get_json_object', return_value={}),\
+            mock.patch('azure.cli.command_modules.keyvault._validators.certificate_type', return_value="fake-cert"):
+        command_loader = load_command_table_and_command_loader(modules,
+                                                               git_source,
+                                                               git_target,
+                                                               git_repo,
+                                                               include_whl_extensions,
+                                                               cli_ci)
+        simple_command_table = simplify_command_table(command_loader.command_table)
+        parser = command_loader.cli_ctx.invocation.parser
+        commands_without_tests = []
+
+        test_exclusions = load_exclusions(exclusion_path)
+        commands_record_file = [get_cli_repo_path(), 'az_command_coverage.txt']
+        for ns in parse_test_commands(parser, commands_record_file):
```

Comment:
why not `os.path.join(get_cli_repo_path(), commands_record_file)` here?

Reply:
Done